pkg/destroy/openstack/openstack.go

```diff
@@ -259,7 +259,7 @@ func deleteServerGroups(opts *clientconfig.ClientOpts, filter Filter, logger log
 		return false, nil
 	}
 
-	allPages, err := servergroups.List(conn).AllPages()
+	allPages, err := servergroups.List(conn, nil).AllPages()
```

Comment:
I would expect the commit message to explain why this change was needed, even perhaps splitting this into a separate commit.

Reply:
I believe the reason is https://github.com/gophercloud/gophercloud/pull/2070

Reply:
Thanks Mike, that's correct. I have updated the commit message with that information.